packages/nextjs/app/builders/0x4a9A95B6fe3b9416f0c78A8735Aa075c75AF46a4/page.tsx

```diff
@@ -0,0 +1,79 @@
+"use client";
+
+import React from "react";
+import Image from "next/image";
+//Icons
+import { FaGithub } from "react-icons/fa";
+import { RiTwitterXFill } from "react-icons/ri";
+import { SiGmail } from "react-icons/si";
+
+const RohanPersonalPage = () => {
+  return (
+    <div className=" flex justify-center items-center">
+      <div>
+        <div className=" pt-24 lg:pt-[100px]">
+          <div className="flex flex-col-reverse lg:flex-row items-center justify-center gap-x-[50px] md:mx-[550px]">
+            <div className=" w-full">
+              <h1 className=" text-[40px] font-bold md:my-[1rem]">Hey! I&apos;m Rohan👋🏼</h1>
+              <p className=" text-[14px] md:text-lg p-1 text-[#e7e5e5] font-semibold w-full text-justify mb-[6px]">
```

Comment:
Instead of hardcoding colors(eg. `text-[#e7e5e5]`), you could use semantic colors eg:(`text-primary-content`) [more details here](https://daisyui.com/docs/colors/) . Scaffold-ETH 2 has nice color themes setup, checkout [`tailwind.config.js`](https://github.com/BuidlGuidl/batch8.buidlguidl.com/blob/main/packages/nextjs/tailwind.config.js)

An advantage of this will be once you use semantic colors, the color will based on user selected dark or light mode. 

Also you could install this extension ["color highlight"](https://marketplace.visualstudio.com/items?itemName=naumovs.color-highlight)  on vscode, which will render the configured colors in `tailwind.config.js` from which you can pick. 

<details>

<summary>
Example : 
</summary>

![Screenshot 2024-07-23 at 5 50 55 PM](https://github.com/user-attachments/assets/b658cb72-aac9-4743-85fa-2492e428470e)


</details>

Reply:
I replaced the hardcoded colors with semantic colors as per the `tailwind.config.js` file.